src/components/PostInfo/PostInfo.scss

```diff
@@ -1 +1,16 @@
-//
+.PostInfo {
+  margin: 10px auto;
+  width: 90%;
+  border: 1px solid #000;
+  border-radius: 8px;
+  background-color: #D3D3D3;
```

Comment:
```suggestion
  background-color: #d3d3d3;
```
it is reason of failed tests on github
![Screenshot from 2023-09-08 12-44-59](https://github.com/mate-academy/react_static-list-of-posts-js/assets/33029864/cbead06d-2980-4f9f-89c2-5a863c6364c7)